core/src/main/scala/wartremover/warts/Contains.scala

```diff
@@ -0,0 +1,34 @@
+package org.wartremover
+package warts
+
+object Contains extends WartTraverser {
+  private[warts] def message: String = "Don't use List or Option `contains` method because is not typesafe"
```

Comment:
actually they _are_ type safe, just not very useful.

Reply:
you can implement something like that `List(1).contains(true)` without any warning about it

Reply:
i agree a warning would be a good thing - my issue is with the wording, because _technically_ it _is_ type safe: you'll never get a `ClassCastException` at run time, the result is correct, and there are even legitimate use cases for this behaviour. just not very many.

Reply:
I agree, can you suggest better wording for the warning?

Reply:
sorry, i'm not even sure i'd want to disable these functions without some more useful replacement - which if it existed, should probably be suggested in the message

Reply:
`contains` in Lists and Options is type unsafe it is possible to implement List(1).contains("1"). Use type-safe implementation for example from cats `List.contains_`

something like that?